include/onnxruntime/core/framework/op_kernel_info.h

```diff
@@ -26,7 +26,6 @@ class OpKernelInfo : public OpNodeProtoHelper<ProtoHelperNodeContext> {
                         const IExecutionProvider& execution_provider,
                         const std::unordered_map<int, OrtValue>& constant_initialized_tensors,
                         const OrtValueNameIdxMap& mlvalue_name_idx_map,
-                        const FuncManager& funcs_mgr,
```

Comment:
Indeed when `funcs_mgr` gets used in this class, it can't be const.  There is hidden const_cast inside. 
FuncManager is stateful. It holds DLLs handles .